pkg/printers/internalversion/printers_test.go

```diff
@@ -1419,139 +1438,37 @@ func TestPrintHumanReadableWithNamespace(t *testing.T) {
 			},
 			isNamespaced: false,
 		},
-		{
-			obj: &unstructured.Unstructured{
-				Object: map[string]interface{}{
-					"kind":       "Foo",
-					"apiVersion": "example.com/v1",
-					"metadata":   map[string]interface{}{"name": "test", "namespace": namespaceName},
-				},
-			},
-			isNamespaced: true,
-		},
-		{
-			obj: &unstructured.Unstructured{
-				Object: map[string]interface{}{
-					"kind":       "Foo",
-					"apiVersion": "example.com/v1",
-					"metadata":   map[string]interface{}{"name": "test"},
-				},
-			},
-			isNamespaced: false,
-		},
 	}
 
+	//*******//
+	options := printers.PrintOptions{
+		WithNamespace: true,
+		NoHeaders:     true,
+	}
+	generator := printers.NewTableGenerator().With(AddHandlers)
+	printer := printers.NewTablePrinter(options)
 	for i, test := range table {
-		printer := printers.NewTablePrinter(printers.PrintOptions{
-			WithNamespace: true,
-			NoHeaders:     true,
-		})
-		AddHandlers(printer)
+		table, err := generator.GenerateTable(test.obj, options)
+		if err != nil {
+			t.Fatalf("An error occurred generating table for object: %#v", err)
+		}
 		buffer := &bytes.Buffer{}
-		err := printer.PrintObj(test.obj, buffer)
+		err = printer.PrintObj(table, buffer)
 		if err != nil {
 			t.Fatalf("An error occurred printing object: %#v", err)
 		}
 		if test.isNamespaced {
-			if !strings.HasPrefix(buffer.String(), namespaceName+" ") {
```

Comment:
curious why this changed...

Reply:
I left a debugging change in the code; this shouldn't be here. Thanks for catching it.